release/Makefile

```diff
@@ -106,7 +106,7 @@ test: manifests generate fmt vet ## Run tests.
 
 ##@ Build
 
-build: clean fmt vet ## Build release binary.
+build: clean fmt ## Build release binary.
```

Comment:
I don't think this was ever vetting correctly
Now that I fixed the vet command, it spits a bunch of errors
I'm leaving that to be fixed later in the future. I suspect those errors have been there for a while